tests/db/jobs/test_jobcontroller.py

```diff
@@ -526,6 +526,9 @@ def test_set_job_run_properties(job_controller, one_job) -> None:
     assert job_controller.set_job_run_properties(resources=qr)
     assert job_controller.get_job_doc(job_id=one_job[0].uuid).resources == qr
 
+    assert job_controller.set_job_run_properties(priority=10)
+    assert job_controller.get_job_doc(job_id=one_job[0].uuid).priority == 10
+
```

Comment:
Maybe check it's not set back to 0 if we don't explicitly set priority in the set_job_run_properties?

```suggestion
    assert job_controller.get_job_doc(job_id=one_job[0].uuid).priority == 10
    assert job_controller.set_job_run_properties()
    assert job_controller.get_job_doc(job_id=one_job[0].uuid).priority == 10
```